mlir/include/mlir/Dialect/Vector/IR/VectorOps.td

```diff
@@ -2183,6 +2185,8 @@ def Vector_CompressStoreOp :
     correspond to those of the `llvm.masked.compressstore`
     [intrinsic](https://llvm.org/docs/LangRef.html#llvm-masked-compressstore-intrinsics).
 
+    Note that the index increment is done conditionally.
```

Comment:
Why copying this from l. 2175 ?

Reply:
Did you mean 
`Note, at the moment this Op is only available for fixed-width vectors.`

Reply:
Rebase gone wrong 🤦🏻 Thanks!